blebox_uniapi/box_types.py

```diff
@@ -149,11 +149,10 @@ def get_latest_api_level(product_type: str) -> Union[dict, int]:
             ],
         }
     },
-    # saunaBox
     "thermoBox": {
         20200229: {
-            "api_path": "/state/extended",
-            "extended_state_path": "/state/extended",
+            "api_path": "/api/thermo/state",
```

Comment:
Those are deprecated according to:
https://technical.blebox.eu/openapi_thermobox/openAPI_thermoBox_20200229.html#tag/Control-and-State/paths/~1api~1thermo~1state/get